13/root/usr/share/container-scripts/postgresql/common.sh

```diff
@@ -308,11 +308,11 @@ run_pgupgrade ()
   if test "$old_raw_version" = 92; then
     old_collection=postgresql92
   else
-    old_collection=rh-postgresql$old_raw_version
+    old_collection=postgresql-$old_raw_version
   fi
```

Comment:
This does not correspond with the current state of PostgreSQL rpm names. We need to adjust it and check if the `old_colllection` variable is used

Reply:
changed